qiskit/qpy/__init__.py

```diff
@@ -105,9 +105,203 @@
 Version 5
 =========
 
-Version 5 changes from :ref:`qpy_version_4` by changing two payloads the INSTRUCTION metadata
-payload and the CUSTOM_INSTRUCTION block. These now have new fields to better account for
-:class:`~.ControlledGate` objects in a circuit.
+Version 5 changes from :ref:`qpy_version_4` by adding support for :class:`.~ScheduleBlock`
+and changing two payloads the INSTRUCTION metadata payload and the CUSTOM_INSTRUCTION block.
+These now have new fields to better account for :class:`~.ControlledGate` objects in a circuit.
+In addition, wew payload MAP_ITEM is defined to implement the :ref:`qpy_mapping` block.
+
+In QPY version 5 and above,
+
+.. code-block:: c
+
+    struct {
+        char type;
+    }
+
+immediately follows the file header block to represent the program type stored in the file.
+
+- When ``type==c``, :class:`~.QuantumCircuit` payload follows
+- When ``type==s``, :class:`~.ScheduleBlock` payload follows
+
+Note that different programs cannot be packed together in the same file.
```

Comment:
Do you want to use a `.. note::` directive here to make it stand out in the rendered doc?